library/dnet.c

```diff
@@ -46,6 +46,29 @@
 
 #include "logger.hpp"
 
+int dnet_remove_local_new(struct dnet_backend_io *backend, struct dnet_node *n, struct dnet_id *id,
+			  void *packet, size_t packet_size)
```

Comment:
Why does it accept raw packet instead of parameters for cas? Calling code shouldn't know about packet format.